runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java

```diff
@@ -56,30 +56,67 @@ public static boolean browse(final String url)
 			return false;
 		}
 
+		if (OSType.getOSType() == OSType.Linux && attemptCommandBrowse("xdg-open " + url))
+		{
+				log.debug("Opened browser through xdg-open to {}", url);
+				return true;
+		}
+
+		if (attemptDesktopBrowse(url))
```

Comment:
You should try desktop browse first, then try xdg-open and if nothing works show the message box